src/index.html

```diff
@@ -1,13 +1,478 @@
+          </a>
+          <a
+            href="tel:+12129233700"
+            class="
+            menu__call-to-order
+            menu__hover-affect
+          "
+          >
+            Call to order
+          </a>
+        </div>
+      </div>
+
+    </aside>
+
+    <main class="main">
+
+      <section class="about-us" id="about-us">
+        <div class="about-us__top">
+          <p class="about-us__top-title">Plan your visit</p>
+          <h2 class="section-title section-title--museum-hours">Museum hours</h2>
+        </div>
+        <div class="about-us__content">
+          <div class="address">
+            <p class="address__title">The Met Fifth Avenue</p>
+            <div class="day-and-time">
+              <div class="day-and-time__time">
+                <p>10 am - 5 pm</p>
+                <p>10 am - 9 pm</p>
+                <p>Closed</p>
+              </div>
+              <div class="day-and-time__days">
+                <p>Sun, Mon, Tue, Thu</p>
+                <p>Fri, Sat</p>
+                <p>Wen</p>
+              </div>
+            </div>
+            <div class="closed-on">
+              <p class="closed-on--1">Closed on</p>
+              <p class="closed-on--2">
+                Thanksgiving Day, Dec 25, Jan 1, and the first Monday in May.
+              </p>
+            </div>
+          </div>
+
+          <div class="address">
+            <p class="address__title address__title--closters">The Met Cloisters</p>
+            <div class="day-and-time">
+              <div class="day-and-time__time">
+                <p>10 am – 4.30 pm</p>
+                <p>Closed</p>
+              </div>
+              <div class="day-and-time__days">
+                <p>Thu - Tue</p>
+                <p>Wen</p>
+              </div>
+            </div>
+            <div class="closed-on">
+              <p class="closed-on--1">Closed on</p>
+              <p class="closed-on--2">Thanksgiving Day, Dec 25, Jan 1</p>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="now-on-view" id="now-on-view">
+        <h2 class="section-title now-on-view__title">Now On View</h2>
+        <p class="now-on-view__2nd-title">Explore exhibition gallery guides, videos, articles, and more.</p>
+
+        <div class="now-on-view__container">
+          <article class="inside-part">
+            <div class="inside-part__photos">
+              <img
+                class="
+                inside-part__photo
+                inside-part__photo-mob--hide
+                inside-part__photo--wide
+                "
+                src="./images/old-masters-desktop-wide.png"
+                alt="old-masters">
+              <img
+                class="
+                inside-part__photo
+                inside-part__photo-tabl--hide
+                inside-part__photo--wide
+                "
+                src="./images/old-masters-square-left.png"
+                alt="old-masters">
```

Comment:
Need to format it everywhere
```suggestion
              <img
                  class="
                    inside-part__photo
                    inside-part__photo-tabl--hide
                    inside-part__photo--wide"
                  src="./images/old-masters-square-left.png"
                  alt="old-masters"
                >
```